SUMMARY.md

```diff
@@ -99,12 +99,14 @@
         * [Retrieving the Rebalance State](api/rebalance-state.md)
         * [Aborting a Rebalance](api/rebalance-deactivation.md)
     * [Data Retrieval APIs](data-retrieval-apis.md)
-        * [Retrieving Raw Data](api/read-raw.md)
-        * [Retrieving Rollup Data](api/read-rollup.md)
-        * [Retrieving Text Data](api/read-text.md)
-        * [Retrieving Histogram Data](api/read-histogram.md)
+        * [Finding Streams by Name or Tags](api/search-tags.md)
+        * [Retrieving Data](api/fetch.md)
         * [Retrieving Graphite-style Data](api/read-graphite.md)
-        * [Searching Tags](api/search-tags.md)
+        * Legacy Single-Stream Retrieval
```

Comment:
Because we're using the chapter-toggle plugin to keep the left menu collapsed, all of the headings ("chapters") need to be actual links, or their sub-items will be inaccessible (though still rendered and able to be pulled up directly).  Consider adding an intermediate page, as was done with `api/legacy-data-submission-apis.md`.

Reply:
but add that page at the top of the tree, as a peer of `data-retrieval-apis.md`, not under `api/`.